File: BuildComputer/BuildComputer/ComputerComponent.cpp
#include "ComputerComponent.h"

ComputerComponent::ComputerComponent()
{

}

string ComputerComponent::printThis()
{
	string display = to_string(price);
	display += "     " + name;
	return display;
}

int ComputerComponent::getPrice()
{
	return price;
}

ComputerComponent::~ComputerComponent()
{
	
}

File: BuildComputerCSharp/BuildComputerCSharp/ComputerComponent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BuildComputerCSharp
{
    class ComputerComponent
    {
        protected string name;
        protected int price;
        public ComputerComponent()
        {

        }
        public virtual string printThis()
        {
            string display = price.ToString();
            display += "     " + name;
            return display;
        }
        public int getPrice()
        {
            return price;
        }
    }
}
